### ISSUE TITLE
"insufficient memory" when using case outside of switch

### ISSUE THREAD

**Issue by AGraber** [Zeex/pawn#574]
### Issue description:
When using a case statement outside of a switch, with enum consts and then returning a function, the compiler triggers an `insufficient memory` error, when it should generate a `invalid statement; not in switch` error.

### Minimal complete verifiable example (MCVE):
```pawn
enum
{
	Test1,
	Test2,
}

main()
{
	case Test1, Test2:
	{
		return TestFunction1();
	}
}

TestFunction1()
{
	return 1;
}
```

### Workspace Information:

* Compiler version: 3.10.10
* Command line arguments provided (or sampctl version):
* Operating System: Windows 10

**Comment by Daniel-Cortez** [comment]
The real problem here is that when there's a case outside of switch, the compiler only handles the `case` keyword, leaving out the case values and the ending `:`, which is why the subsequent `Test1, Test2` and `: { return TestFunction1();` in the above example are treated as separate statements, causing multiple errors (which aren't displayed at the 1'st compilation pass) and ending with a fatal error.

**Comment by Daniel-Cortez** [comment]
The problem has been fixed in #599, so we should probably close this issue.
@pawn-lang @AGraber
